RELEASE/scripts/autoscend.ash

```diff
@@ -2871,7 +2871,7 @@ void auto_begin()
 	auto_log_info("This is version: " + svn_info("autoscend").last_changed_rev + " Mafia: " + get_revision());
 	auto_log_info("This is day " + my_daycount() + ".");
 	auto_log_info("Turns played: " + my_turncount() + " current adventures: " + my_adventures());
-	auto_log_info("Current Ascension: " + auto_my_path());
+	auto_log_info("Current Ascension: " + my_path().to_string());
```

Comment:
.to_string() is not necessary for any of the other types when composting into a string. so it is highly likely that it won't be necessary for my_path() either
we will know when it is actually released though

Reply:
ah, I wasn't sure. I removed the offenders where I found them.